octobot_trading/personal_data/orders/channel/orders_updater.py

```diff
@@ -107,6 +107,7 @@ async def fetch_and_push(self, is_from_bot=True, limit=ORDERS_UPDATE_LIMIT, retr
         finally:
             if self.channel is not None:
                 self.channel.exchange_manager.exchange_personal_data.orders_manager.are_exchange_orders_initialized = True
+                self.channel.exchange_manager.exchange_personal_data.portfolio_manager.enable_portfolio_available_update_from_order = True
```

Comment:
I'm not sure it's the proper place to set a portfolio_manager flag. According to the architecture it should be set in the "portfolio" channel